templates/al2023/runtime/rootfs/etc/systemd/system/setup-local-disks.service

```diff
@@ -4,7 +4,8 @@ Before=containerd.service kubelet.service
 
 [Service]
 Type=oneshot
-ExecStart=/usr/bin/setup-local-disks raid0
+Environment=LOCAL_DISK_STRATEGY=none
+ExecStart=/usr/bin/setup-local-disks $LOCAL_DISK_STRATEGY
```

Comment:
sounds like we are gonna do another `EnvironmentFile` created via nodeadm?

Reply:
Yep!